apps/api/src/app/auth/usecases/register/user-register.usecase.ts

```diff
@@ -1,15 +1,16 @@
 import { Inject, Injectable } from '@nestjs/common';
 import { OrganizationEntity, UserRepository } from '@novu/dal';
 import * as bcrypt from 'bcrypt';
+import { SignUpOriginEnum } from '@novu/shared';
+import { AnalyticsService } from '@novu/application-generic';
```

Comment:
Some files have just imports reordered, when something more important will be I'll left a comment